examples/fillform.js

```diff
@@ -1,29 +1,30 @@
-import { check } from 'k6';
-import launcher from 'k6/x/browser';
+import { check } from "k6";
+import launcher from "k6/x/browser";
```

Comment:
These changes are out of place for this PR, so please leave them out and create an issue so that we can standardize this across all JS files.